.circleci/config.yml

```diff
@@ -385,27 +415,15 @@ jobs:
           name: "Run Cypress Integration Tests"
           command: "cd tdrs-frontend; yarn cy:run:ci"
       - run:
-          name:  "Test: Execute Frontend  OWASP ZAP Vulnerability Scan"
+          name: "Test: Build and Spin-up React app behind Nginx"
+          command: "cd tdrs-frontend; docker-compose up -d --build"
+      - run:
+          name:  "Test: Execute Frontend OWASP ZAP Vulnerability Scan"
           command: "./scripts/zap-scanner.sh frontend circle"
       - store_artifacts:
           path: tdrs-frontend/reports/owasp_report.html
       - store_artifacts:
-          path: tdrs-frontend/pa11y-screenshots/homepage.png
-
-      - store_artifacts:
-          path: tdrs-frontend/pa11y-screenshots/edit-profile.png
-
-      - store_artifacts:
-          path: tdrs-frontend/pa11y-screenshots/request.png
-
-      - store_artifacts:
-          path: tdrs-frontend/pa11y-screenshots/login.png
-
-      - store_artifacts:
-          path: tdrs-frontend/pa11y-screenshots/data-files.png
-
-      - store_artifacts:
-          path: tdrs-frontend/pa11y-screenshots/welcome.png
+          path: tdrs-frontend/pa11y-screenshots/
```

Comment:
This change isn't really related to this work, but if we pass the whole directory here we don't have to keep defining every pa11y screenshot.